src/mpyl/constants.py

```diff
@@ -5,5 +5,8 @@
 DEFAULT_RUN_PROPERTIES_FILE_NAME = "run_properties.yml"
 
 PR_NUMBER_PLACEHOLDER = "{PR-NUMBER}"
+SERVICE_NAME_PLACEHOLDER = "{SERVICE-NAME}"
```

Comment:
Interesting that these weren't constants already 😅

Reply:
I guess because they were only used in 1 place, but yeah